sentinel-transport/sentinel-transport-netty-http/src/main/java/com/alibaba/csp/sentinel/transport/heartbeat/HttpHeartbeatSender.java

```diff
@@ -104,7 +106,8 @@ public HttpHeartbeatSender() {
 
     @Override
     public boolean sendHeartbeat() throws Exception {
-        if (StringUtil.isEmpty(consoleHost)) {
+        if (StringUtil.isEmpty(consoleHost) || invalidPort(consolePort)) {
+            RecordLog.warn("[HttpHeartbeatSender] Failed to send heartbeat for invalid args consoleHost:{0} consolePort:{1}", consoleHost, consolePort);
```

Comment:
Maybe it's not necessary to log here, as we also checked and warned in the constructor of the HeartbeatSender impls.